src/python/pants_release/generate_github_workflows.py

```diff
@@ -1234,8 +1234,7 @@ def release_jobs_and_inputs() -> tuple[Jobs, dict[str, Any]]:
                     "name": "Get release notes",
                     "run": dedent(
                         """\
-                        REF="${{ needs.release_info.outputs.build-ref }}"
-                        ./pants run src/python/pants_release/get_release_notes.py -- ${REF#"release_"} > notes.txt
+                        ./pants run src/python/pants_release/generate_github_workflows.py -- "${{ needs.release_info.outputs.build-ref }}" > notes.txt
```

Comment:
This looks like the script is running itself. Should it be running

```suggestion
                        ./pants run src/python/pants_release/changelog.py -- "${{ needs.release_info.outputs.build-ref }}" > notes.txt
```

Reply:
Wouldnt be a Josh GHA PR without dump typos